src/css/custom.css

```diff
@@ -16,6 +16,11 @@
   --ifm-color-primary-lightest: rgb(146, 224, 208);
   --ifm-code-font-size: 95%;
   --doc-sidebar-width: 15em !important;
+  --ifm-container-width: 100vw;
+}
+
+.container.docItemWrapper_node_modules-\@docusaurus-theme-classic-lib-next-theme-DocPage- {
```

Comment:
Saying this as someone who knows no CSS: is it worth it to comment what our custom CSS is doing?

Reply:
Also good point. Created #56